docs/datasets.md

```diff
@@ -3751,8 +3751,10 @@ Year      = {2014}}
 
 The EMNIST dataset is a set of handwritten character digits derived from the NIST Special Database 19 and converted to a 28x28 pixel image format and dataset structure that directly matches the MNIST dataset.
 
-* URL: [https://www.nist.gov/node/1298471/emnist-dataset](https://www.nist.gov/node/1298471/emnist-dataset)
-* `DatasetBuilder`: [`tfds.image.mnist.EMNIST`](https://github.com/tensorflow/datasets/tree/master/tensorflow_datasets/image/mnist.py)
+*   URL:
```

Comment:
This file generated automatically with `document_datasets.py`. And it's not related to Fruits 360 dataset.

Reply:
Please delete this changes.

Reply:
I think it's done when delete the last commit. You can rollback git and force the push.

Reply:
I actually used `document_datasets.py` to generate it, which is why I was surprised it conflicted and had me merge it. I'll roll it back and try again.